source/block-and-log-streamer.ts

```diff
@@ -11,7 +11,7 @@ import * as createUuid from "uuid";
 
 export interface Configuration {
 	/** number of blocks to retain in history, defaults to 100 */
-	blockRetention?: number
+	blockRetention?: number;
```

Comment:
In my newer projects I have switched to "no semicolons" for TypeScript, but I believe this project still uses semicolons.  I wouldn't be against moving to no-semicolons project wide, but I'm also fine with sticking with the status quo in this project and adding semicolons where they are missing.

Reply:
I was able to easily switch to no semi-colons.